packaging/examples/metrics/kafka-connect-metrics.yaml

```diff
@@ -47,10 +47,18 @@ data:
         clientId: "$2"
         $3: "$4"
       help: "Kafka $1 JMX metric info version and commit-id"
-      type: GAUGE
+      type: UNTYPED
 
     #kafka.consumer:type=consumer-fetch-manager-metrics,client-id="{clientid}",topic="{topic}"", partition="{partition}"
-    - pattern: kafka.consumer<type=consumer-fetch-manager-metrics, client-id=(.+), topic=(.+), partition=(.+)><>(.+-total|compression-rate|.+-avg|.+-replica|.+-lag|.+-lead)
+    - pattern: kafka.consumer<type=consumer-fetch-manager-metrics, client-id=(.+), topic=(.+), partition=(.+)><>(.+-total)
+      name: kafka_consumer_fetch_manager_$4
+      labels:
+        clientId: "$1"
+        topic: "$2"
+        partition: "$3"
+      help: "Kafka Consumer JMX metric type consumer-fetch-manager-metrics"
+      type: COUNTER
```

Comment:
general question applying to all the others ...
Is the change from GAUGE to COUNTER related to OpenMetrics or the JMX update? I didn't get it from the description, even because GAUGE can increase and decrease while a COUNTER can just increase so their behaviour is totally different. Can you elaborate a little bit more please?

Reply:
Yes. The thing is that only counters can have the `_total` suffix. So as long as it is a gauge, the JMX Exporter would remove it and change the metric names. So I had to change these to counters to keep the name.

Reply:
So just for double checking, I would assume that those `_total` metrics coming from Kafka were already to be considered COUNTER(s) before (because just increasing) and we made a mistake to "translate" them as GAUGE. Is this assumption right? Wdyt?

Reply:
I think so ... those used in our dashabords are treated as counters. But there are so many metrics with things I never heard about that it is really hard to say it with certainty for all of them. If anyone wants to dissect them in detail, it can be done in a separate PR.

The thing is, that the older versions of the JMX Exporter did not really care about the type. So it didn't matter if it is gauge or counter. You were able to name it any way you wanted with any type. The current version validates the metric names according to the type. S you have to use the matching type for the name. (well, the INFO type is not supported, so those metric names had to change)

Reply:
So we are kind of relying and trusting the Kafka contributors to name metrics right based on the types. Got it. Thanks.